### PR TITLE
doc: broken links in config overrides

### PR DESCRIPTION
**Pull request by javiermolinar**
**What this PR does**:
- Fix some broken links related with config overrides

**Checklist**
- [X] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -49,8 +49,8 @@
 * [ENHANCEMENT] Added new Traces api V2[#3912](https://github.com/grafana/tempo/pull/3912) (@javiermolinar)
 * [ENHANCEMENT] Update to the latest dskit [#3915](https://github.com/grafana/tempo/pull/3915) (@andreasgerstmayr)
 * [ENHANCEMENT] Reduce allocs building queriers sharded requests [#3932](https://github.com/grafana/tempo/pull/3932) (@javiermolinar)
-
 * [ENHANCEMENT] Implement polling tenants concurrently [#3647](https://github.com/grafana/tempo/pull/3647) (@zalegrala)
+* [ENHANCEMENT] Fix broken links in the documentation [#3956](https://github.com/grafana/tempo/pull/3956) (@javiermolinar)
```
Comment (joe-elliott) [inline CHANGELOG.md:53]: we normally don't do changelog entries for the docs, but i'm not opposed to it. if we do start this i think we should mark them separately tho.

`[DOC] Fix broken lines ... `

Comment (javiermolinar) [reply CHANGELOG.md:53]: ah didn't know, I can remove it

**Review by knylander-grafana — APPROVED** [review]
Thank you for fixing broken links.

**Comment by javiermolinar** [comment]
@knylander-grafana I think is safe to merge this. I don't have that power yet :P

**Review by knylander-grafana — APPROVED** [review]

**Pull request merged**